imap/http_dav.c

```diff
@@ -2724,10 +2727,16 @@ int propfind_curprivset(const xmlChar *name, xmlNsPtr ns,
                     flags = PRIV_INBOX;
                 else if (!strcmp(fctx->req_tgt->collection, SCHED_OUTBOX))
                     flags = PRIV_OUTBOX;
+
+                flags |= PRIV_NOSUBCOL;
             }
         }
+        else if (fctx->req_tgt->collection &&
+                 fctx->req_tgt->namespace->id == URL_NS_ADDRESSBOOK) {
+            flags |= PRIV_NOSUBCOL;
+        }
 
-        flags += PRIV_CONTAINED;
+        flags |= PRIV_CONTAINED;
```

Comment:
Nice that this just works unless you accidentally do it twice.  Ouch!  And even then it still works just fine so long as you don't have a bigger key :p